package.json

```diff
@@ -79,44 +79,46 @@
     "netlify-cms": "^2.10.55",
     "node-sass": "^4.14.1",
     "normalize-scss": "^7.0.1",
+    "npm": "^6.14.10",
```

Comment:
Why do we need NPM?

Reply:
May be there is not more Important as we have already global npm installed. we can remove it.